### PR TITLE
EB-21 CA - Structure  create structure in CA folder and 2 hmtl file a…

### PR DESCRIPTION
**Pull request by 1986-OldMan**
…dded in pages folder,singin.html and singup.html

### CODE REVIEW

**Comment by LabiciIvan** [comment]
@1986-OldMan  Please use the instruction guide on how to format the commit message [Exchange_Bot How ? Branch_name | Commit_message_title](https://github.com/LabiciIvan/exchange-bot/wiki/Exchange_Bot-How-%3F--Branch_name-%7C-Commit_message_title)
Your commit message looks like this **EB-21 CA - Structure create structure in CA folder and 2 hmtl file a… #23**
But it should be  **EB-21: CA - Structure #21**

**Review by LabiciIvan — CHANGES_REQUESTED** [review]
Why do you have changes in PRO ? 
You work on a CA task, you should have only changed files that are in CA folder.

**Review by LabiciIvan — COMMENTED** [review]
@1986-OldMan  Please use the instruction guide on how to format the commit message [Exchange_Bot How ? Branch_name | Commit_message_title](https://github.com/LabiciIvan/exchange-bot/wiki/Exchange_Bot-How-%3F--Branch_name-%7C-Commit_message_title)
Your commit message looks like this **EB-21 CA - Structure create structure in CA folder and 2 hmtl file a… #23**
But it should be  **EB-21: CA - Structure #21**

**Comment by 1986-OldMan** [comment]
There are changes in PRO , because I have to uncomment on two lines of javascript code that you told me about.

Line 9 and line 21 of the code in PRO/app.js.

**Comment by 1986-OldMan** [comment]
Ok ,I will look over the instructions one more time and redo it again.

**Comment by LabiciIvan** [comment]
You only need to uncomment line **9** and line **21** of code in **Pro/app.js** just when you want to make tables in your database, otherwise you should keep it commented.

File: PRO/app.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ let logger = require('morgan');
 
 let usersRouter = require('./routes/users');
 let exchangesRouter = require('./routes/exchanges');
-// let migrateRouter = require('./middleware/migrateBase');
+let migrateRouter = require('./middleware/migrateBase');
```
Comment (LabiciIvan) [inline PRO/app.js:9]: Comment this line, almost most of the time we will let it be commented.

File: PRO/app.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'public')));
 
-// app.use('/migrate', migrateRouter);
+app.use('/migrate', migrateRouter);
```
Comment (LabiciIvan) [inline PRO/app.js:21]: And this as well, keep it commented and only uncomment if you want to migrate database locally.

**Pull request closed without merging**